GATK_QuickByte.md

```diff
@@ -333,23 +374,6 @@ Here is a sample PBS script combining everything we have above, with as much par
 			-O $src/bams/{}_dedup.bam
 		rm $src/alignments/{}.sam'
 
```

Comment:
Shouldn't reference indexing be included?

bwa index -p $reference ${reference}.fa
samtools faidx ${reference}.fa -o ${reference}.fa.fai
picard CreateSequenceDictionary \
   		R=${reference}.fa \
   		O=${reference}.dict

Reply:
Added!